README.md

```diff
@@ -45,7 +45,10 @@ Replace contents of "C:\Program Files\Docker Toolbox\start.sh" with this
 ```bash
 #!/bin/bash
 
-export PATH="$PATH:/mnt/c/Program Files (x86)/VMware/VMware Workstation"
+# Uncomment this to enable debug output:
+# export MACHINE_DEBUG="TRUE"
+
+export PATH="$PATH:/c/Program Files (x86)/VMware/VMware Workstation"
```

Comment:
Is the old mount path no longer valid?

Reply:
yes, No /mnt

Reply:
But is `/mnt/c/etc` no longer valid?

Reply:
`/etc` I don't know where your path is this?
Where did you see that `/etc`?
`/c/Program Files (x86)/VMware/VMware Workstation` is the default installation path for VMware Workstation under the Windows system.